Anature/src/application/anatures/moves/moves/SkipTurn.java

```diff
@@ -8,6 +8,7 @@
 	@Override
 	public void activateMove(IAnature source, IAnature target)
 	{
+		// TODO Do we need this?
```

Comment:
I have no idea. I think SkipTurn was added in for the first version of the statuses (for effects like paralysis, sleep, or frozen), but with the new system, I don't think we need it anymore.

Reply:
Ok I removed that println statement